### ISSUE TITLE
city A is buying wheat for the wrong amount

### ISSUE THREAD

**Issue by cenheidlon** [GingaaBread/Unity-Game-Jam-2022#146]
**Describe the bug**
City A is buying wheat for 10 coins when they should be buying it for 12 coins. It is also buying flowers for the wrong amount (14 coins instead of 16 coins).
**EDIT:** also, when I sold wheat to city A, my total coins only went up by 5 (lmk if I should open a separate issue for this)

**To Reproduce**
1. Sell wheat to city A.
2. receive 10 coins

**Expected behavior**
City A should buy wheat (and all other products) for two coins more than their usual selling price. (12 for wheat, 26 for oats, 16 for flowers, 16 for rice)

**Screenshots**
![Screenshot 2022-11-28 at 17 11 08](https://user-images.githubusercontent.com/119008588/204391747-58e61ee5-5121-49bc-9e3c-db13dca45a03.png)
![Screenshot 2022-11-28 at 17 15 01](https://user-images.githubusercontent.com/119008588/204392440-8ce27ae4-1a7a-4bad-91bf-aa7f8f3fc7ae.png)

**Desktop (please complete the following information):**
 - Browser [chrome/brave]

**Comment by ballardi** [comment]
should be fixed once this is merged and released
https://github.com/GingaaBread/Unity-Game-Jam-2022/pull/158

**Comment by ballardi** [comment]
no need to make separate ticket about the $5 issue - that should be resolved too
